api-tester/build.gradle

```diff
@@ -1,6 +1,11 @@
 apply from: "$rootProject.projectDir/base-application.gradle"
 
 android {
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
+        main.java.srcDirs += 'src/main/java'
+        test.java.srcDirs += 'src/test/java'
+    }
```

Comment:
Hmm is this needed?

Reply:
Maybe not... I added it because it was removed in https://github.com/RevenueCat/purchases-android/pull/616

But yes, it might not be needed, this is probably the default anyway right?

Reply:
I would think so yeah 👍